### PR TITLE
docs: Update Documentation

### PR DESCRIPTION
**Pull request by sushantbasak**
This PR fixes Expired Links in Docs.
Related
#8273 

## Checklist

- [ ] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by dhmlau** [comment]
@sushantbasak, thanks for the PR. Could you please sign the DCO? 
I believe our [commit message guidelines](https://loopback.io/doc/en/lb4/code-contrib-lb4.html#commit-message-guidelines) expects the commit message to begin with lower case, i.e. 
```
docs: fix DataSource Documentation Link
```

instead of 
```
docs: Fix DataSource Documentation Link
```
Otherwise the commit linter will give an error. 

After those are taken care of, squash the commits, it should be good to go. Thanks.

File: docs/site/Creating-datasource-runtime.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ app.dataSource(bookDs, dsName);
 ```
 
 For details about datasource options, refer to the
-[DataSource documentation](https://apidocs.strongloop.com/loopback-datasource-juggler/#datasource).
+[DataSource documentation](http://apidocs.loopback.io/loopback-datasource-juggler/#datasource).
```
Comment (nabdelgadir) [inline docs/site/Creating-datasource-runtime.md:27]:
```suggestion
[DataSource documentation](https://apidocs.loopback.io/loopback-datasource-juggler/#datasource).
```

File: docs/site/guides/data-access/transactions.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ method of `DataSource` class.
 ## Handling Transactions
 
 See
-the [API reference](https://apidocs.strongloop.com/loopback-datasource-juggler/#datasource-prototype-begintransaction) for
+the [API reference](http://apidocs.loopback.io/loopback-datasource-juggler/#datasource) for
```
Comment (nabdelgadir) [inline docs/site/guides/data-access/transactions.md:56]:
```suggestion
the [API reference](https://apidocs.loopback.io/loopback-datasource-juggler/#datasource) for
```

**Review by nabdelgadir — COMMENTED** [review]
Along with @dhmlau's comment, can you also update the links?

File: docs/site/Creating-datasource-runtime.md (hunk 1 above)
Comment (sushantbasak) [reply docs/site/Creating-datasource-runtime.md:27]: Yeah  sure. Will update this PR.

**Comment by sushantbasak** [comment]
I have updated the changes that are requested.

File: docs/site/guides/data-access/transactions.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ method of `DataSource` class.
 ## Handling Transactions
 
 See
-the [API reference](https://apidocs.strongloop.com/loopback-datasource-juggler/#datasource-prototype-begintransaction) for
+the [API reference](https://apidocs.loopback.io/loopback-datasource-juggler/#datasource) for
```
Comment (dhmlau) [inline docs/site/guides/data-access/transactions.md:56]: Let's keep the "begintransaction" portion
```suggestion
the [API reference](https://apidocs.loopback.io/loopback-datasource-juggler/#datasource-prototype-begintransaction) for
```

**Comment by dhmlau** [comment]
@sushantbasak, thanks for incorporating the feedback. I have one minor change (as suggested change). 
Also, could you please sign the DCO? Without signing DCO, we won't be able to land your PR.  Thanks.
You can sign it by running the following command:
```
git commit --amend -s
```

**Comment by sushantbasak** [comment]
@dhmlau For DCO do I need to run **git commit --amend -s** after creating commits?

**Comment by achrinza** [comment]
`-s` is for adding the DCO sign-off; Though the sign-off can be typed manually as well.

`--amend` is for modifying the previous commit (e.g. If you've forgotten to use `-s` for your last commit).

**Comment by sushantbasak** [comment]
I am not able to get why DCO is not working. Can anyone help?

**Comment by dhmlau** [comment]
I think it's because you have 3 commits and 2 of them are signed. Could you please squash the commits, and check that the resulting one commit is signed?

**Comment by sushantbasak** [comment]
**This is the git history**
image
**I am getting this when I am trying to rebase.**
image

Can you let me know what should I do after this?

**Comment by dhmlau** [comment]
Closing as abandon. @sushantbasak, if you decide to continue with this PR, please sign your commits with DCO. Thanks.

**Pull request closed without merging**